torchfix/visitors/deprecated_symbols/size_average.py

```diff
@@ -0,0 +1,60 @@
+"""size_average and reduce are deprecated, please use reduction='mean' instead."""
+
+import libcst as cst
+from ...common import TorchVisitor, get_module_name
+from torch.nn._reduction import legacy_get_string
```

Comment:
Right now TorchFix doesn't depend on or use PyTorch.
Maybe we can add PyTorch to CI, but then which version?
Is it hard to copy or re-implement `legacy_get_string`?